linux_os/guide/services/ntp/chronyd_specify_remote_server/bash/shared.sh

```diff
@@ -0,0 +1,9 @@
+# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol
+. /usr/share/scap-security-guide/remediation_functions
+populate var_multiple_time_servers
+
+config_file="/etc/chrony.conf"
+
+if ! grep -q '^[\s]*server[\s]+[\w]+' "$config_file" ; then
+  {{{ bash_ensure_there_are_servers_in_ntp_compatible_config_file("$config_file", "$var_multiple_time_servers") | indent(2) }}}
```

Comment:
This macro also adds the `iburst` option, which is not mentioned in rule description.

Reply:
I removed the iburst from macro, no rule which is using it is checking for it nor it is mentioned in the description / ocil